reference/datetime/functions/date.xml

```diff
@@ -37,6 +37,15 @@
     <listitem>
      <para>
       Formato aceito pelo <methodname>DateTimeInterface::format</methodname>.
+      <note>
+       <simpara>
+        <function>date</function> sempre gerará
+        <literal>000000</literal> como microssegundos pois requer um parâmetro
```

Comment:
Aqui acredito ser "como" mesmo, veja o [original](https://github.com/php/doc-en/blob/f5b15841e613d09971de3652a8d25da4438cb4a2/reference/datetime/functions/date.xml#L44):

```xml
<literal>000000</literal> as microseconds since it takes an <type>int</type>
```

O que acha?

Reply:
Ah sim. "Como" mesmo, ou ainda "nos". Talvez ainda passar microssegundos para antes dos zeros. "date() sempre gerará microssegundos 000000 pois..."